src/src/pages/capabilities/MyCapabilities.js

```diff
@@ -68,6 +70,89 @@ export default function MyCapabilities() {
   useEffect(() => {
     setShowAwsResourcesSpinner(isLoadingAwsResourcesCounts);
   }, [isLoadingAwsResourcesCounts]);
+
+
+  const columns = useMemo(
+    () => [
+      {
+        accessorFn: (row) => { return { name: row.name, description: row.description } },
+        header: 'Name',
+        size: 350,
+        enableColumnFilterModes: false,
+        Cell: ({ cell }) => {
+          return <div> <Text styledAs="action" as={"div"}>
+            {cell.getValue().name}
+          </Text>
+            <Text styledAs="caption" as={"div"}>
+              {cell.getValue().description}
+            </Text></div>
+        }
+
+      },
+      {
+        accessorKey: 'awsResources',
+        header: 'Aws Resources',
+        size: 150,
+        enableColumnFilterModes: false,
+        muiTableHeadCellProps: {
+          align: 'center'
+        },
+        muiTableBodyCellProps: {
+          align: 'center'
+        }
+        
+
+      },
+      {
+        accessorKey: 'id',
+        header: 'Costs',
+        size: 150,
+        enableColumnFilterModes: false,
+        muiTableHeadCellProps: {
+          align: 'center'
+        },
+        muiTableBodyCellProps: {
+          align: 'right'
+        },
+        Cell: ({ cell }) => {
+          return <div className={styles.costs}>
+            <CapabilityCostSummary
+              data={metricsWrapper.getCostsForCapability(
+                cell.getValue().toLocaleString(),
+                7,
+              )}
+            />
+            <ChevronRight />
+          </div>
+        }
+      },
+    ],
+    [],
+  )
+
+  useEffect(() => {
+
+    const tableData = items.map((item) => {
+
+      const copy = { ...item };
+
+      const awsResources = metricsWrapper.getAwsResourcesTotalCountForCapability(copy.id)
+        ? metricsWrapper.getAwsResourcesTotalCountForCapability(copy.id)
+        : 0;
+
+      copy.awsResources = awsResources;
+
+      return copy;
+
+    });
+
+
+
+    setFullTableData(tableData);
+
+  }, [isLoading, appStatus, isLoadingCosts])
```

Comment:
guess we also need `isLoadingAwsResourcesCounts` here or solve this issue in a nicer way (figure out how to managing loading of capability metrics non-boolean way)

Reply:
If we want to show spinners for individual parts, then I am afraid we will need boolean flags for each such part.
For now our amount of flags is fairly manageable, but if it grows too big let us look at this.

Crazy thought of the day; Having a single key-value map of listeners, triggering on changes, and dispatching calls to the relevant entities -- though I am not sure this is easier to maintain 😅